### PR TITLE
clean up pipelines based on feedback

### PR DESCRIPTION
**Pull request by sbawaska**
incorporate feedback from: https://github.com/pivotal/go-ape/pull/11

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -9,51 +9,47 @@ variables:
...
-    vmImage: $(imageName)
+strategy:
+  matrix:
+    linux:
+      imageName: 'ubuntu-16.04'
+    mac:
+      imageName: 'macos-10.13'
+    windows:
+      imageName: 'vs2017-win2016'
+pool:
```
Comment (scothis) [inline azure-pipelines.yml:21]: nit add a leading newline, since other top level keys also have a newline gap

File: azure-pipelines.yml [hunk 2]
```diff
@@ -9,51 +9,47 @@ variables:
...
-      shopt -s dotglob
-      mv !(gopath) '$(modulePath)'
-      echo '##vso[task.prependpath]$(GOBIN)'
-      echo '##vso[task.prependpath]$(GOROOT)/bin'
-    displayName: 'Set up the Go workspace'
+steps:
+- task: GoTool@0
+  inputs:
+    version: 1.12
+- bash: |
```
Comment (scothis) [inline azure-pipelines.yml:29]: nit: add a leading newline since other steps are separated by newlines

**Review by scothis — APPROVED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (sbawaska) [reply azure-pipelines.yml:21]: done.

File: azure-pipelines.yml (hunk 2 above)
Comment (sbawaska) [reply azure-pipelines.yml:29]: done.

**Pull request merged**